izanami-frontend/src/components/RightSelector.tsx

```diff
@@ -491,14 +495,29 @@ export function RightSelector(props: {
             </>
           )}
         </>
-        {!props.tenant &&
-          !creating &&
-          selectedTenants.length < tenantQuery.data.length && (
+        {!creating &&
+          selectedTenants.length < tenants.length &&
+          !(
+            props.tenant &&
+            defaultValue?.tenants &&
+            props.tenant in defaultValue.tenants
+          ) && (
             <button
               className="btn btn-primary mt-2"
-              onClick={() => setCreating(true)}
+              onClick={() => {
+                setCreating(true);
```

Comment:
I don't think that it's necessary to setCreating(true) in every cases, and immediatly set it to false if props.tenant is set, this should probably be in a else statement

Reply:
Okay, I will modify it